examples/mugen/retrieval/video_clip.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import torch
+from torch import nn
+from torchmultimodal.utils.common import get_current_device
+from transformers import DistilBertConfig, DistilBertModel, DistilBertTokenizer
+
+
+class TextEncoder(nn.Module):
+    """Encode text to a fixed-length vector
+
+    Taken from MUGEN's text encoder
+        (https://github.com/mugen-org/MUGEN_baseline/blob/main/lib/models/videoclip/modules.py)
+    """
+
+    def __init__(
+        self,
+        model_name="distilbert-base-uncased",
+        pretrained=True,
+        max_length=200,
+    ):
+        super().__init__()
+        self.max_length = max_length
+        self.tokenizer = DistilBertTokenizer.from_pretrained(model_name)
+        if pretrained:
+            self.model = DistilBertModel.from_pretrained(model_name)
```

Comment:
non blocking comment: We should align with torchtext on the timeline of landing this model in parallel.